edb/edgeql/compiler/config.py

```diff
@@ -146,7 +146,24 @@ def compile_ConfigReset(
             span=expr.span,
         )
 
-    elif isinstance(info.param_type, s_objtypes.ObjectType):
+    if (
+        expr.scope == qltypes.ConfigScope.INSTANCE and
+        info.backend_setting == 'default_transaction_isolation'
+    ):
+        # Special case for default_transaction_isolation:
+        # Gel default is `serializable`, while PG default is `read committed`.
```

Comment:
Call this out as a `HACK` in the comments.
Some day we may want this to be a general mechanism?
(Not today.)

Reply:
Yeah, now that I'm thinking more about it, maybe we should *always* reset the system backend settings to their Gel defaults explicitly, as we cannot rely on the backend defaults, like users could put anything in `postgresql.conf` and run Gel with `--backend-dsn`.